src/Types.cpp

```diff
@@ -64,4 +67,24 @@ bool Variant::isEmpty() const
     return msg_.isEmpty();
 }
 
+void UnixFd::close()
+{
+    if (fd_ >= 0) {
+        ::close(fd_);
+    }
+}
+
+int UnixFd::checked_dup(int fd)
```

Comment:
Style convention appears to be:
```suggestion
int UnixFd::checkedDup(int fd)
```